_posts/blog/2018-10-17-ReviewGarcíaMárqu

```diff
@@ -0,0 +1,63 @@
+ 
+#Gabriel García Márquez. Harry Ransom Center. The University of Texas. Web.16 de Octubre 2018.https:[//hrc.contentdm.oclc.org/digital/collection/p15878coll51/]
+
+
+Así de extraño como el nacimiento del último de los Buendía que nació con una cola de cerdo en la famosa novela 100 años de Soledad del colombiano Gabriel García Márquez, es extraño para muchos de sus paisanos que el archivo privado de este escritor, premio Nobel de literatura 1982, reposen en el Centro de Archivos Harry Ransom Center en la Universidad de Texas en Austin, en los Estados Unidos. País que vetó por muchos años al escritor por la relación de éste con Fidel Castro y por ser catalogado por muchos como comunista.
+
+Pero por qué la familia del escritor querría que sus archivos, los originales de sus novelas más famosas, sus notas, sus memorias, sus fotos y en fin lo que condensa parte su vida privada y que lo hizo famoso públicamente como escritor estuviese tan lejos de su macondiana tierra?...
```

Comment:
The use of ellipses (...) can be an effective stylistic choice, but it should be used judiciously. Otherwise it has the effect of looking overly informal. Please choose one or two places, and remove the rest.